tests/component-tests/textToSpeech/js/TextToSpeechTests.js

```diff
@@ -11,13 +11,40 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
 */
 
 // Declare dependencies
-/* global fluid, jqUnit, speechSynthesis */
+/* global fluid, jqUnit, speechSynthesis, SpeechSynthesisUtterance */
 
 (function () {
     "use strict";
 
     fluid.registerNamespace("fluid.tests");
 
+    // Ensures that TTS is supported in the browser, including cases wehre the
+    // feature is detected, but where the underlying audio engine is missing.
+    // For example VM's on SauceLabs.
+    //
+    // returns a promise
```

Comment:
I did push a change to the comment, could you let me know how it is now. I'll look into moving it into the main implementation if the comment is okay.